pkg/log/log.go

```diff
@@ -0,0 +1,54 @@
+package log
+
+import (
+	"sync"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Wrapper wrapper logrus logger to enable global configuration
+type wrapper struct {
+	log *logrus.Entry
+}
+
+var logger *wrapper
+var once sync.Once
+
+// New takes in logrus.Entry and configures global logger
+func New(entry *logrus.Entry) {
```

Comment:
This seems like an inflexible interface for configuring a logger. Usually at the top level you start with a logger and a couple of known fields (eg. subscription ID) and as you progress down the rabbit hole you want to be able to inject more (eg. resource group) and more (eg. scale set name).

Reply:
We need requirements form MSFT on this. As per the current understanding, I was under impression that RP will provide its own `entry/logrus` config and it will apply to this particular call. Do we need this for our own use?